rfc/6/README.md

```diff
@@ -76,12 +76,12 @@ Cluster administrator
     - waits for `m0mkfs` to terminate;
     - starts `m0d` process(es).
 
-11. Starts S3 servers.  For each S3 server:
+11. Starts rgw servers.  For each rgw server:
```

Comment:
We do not start RGW or any other client as part of bootstrap now. so lets remove this section altogether

Reply:
Removed